agr_java_core/src/main/java/org/alliancegenome/core/variant/service/AlleleVariantIndexService.java

```diff
@@ -1,78 +1,78 @@
 package org.alliancegenome.core.variant.service;
 
-import com.fasterxml.jackson.core.JsonParser;
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.DeserializationFeature;
-import com.fasterxml.jackson.databind.MapperFeature;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.google.gson.Gson;
+import static org.alliancegenome.core.config.Constants.ES_INDEX;
+
+import java.io.IOException;
+import java.util.*;
+
 import org.alliancegenome.api.entity.AlleleVariantSequence;
 import org.alliancegenome.es.util.EsClientFactory;
 import org.alliancegenome.neo4j.entity.node.*;
-import org.elasticsearch.action.search.SearchRequest;
-import org.elasticsearch.action.search.SearchResponse;
+import org.elasticsearch.action.search.*;
 import org.elasticsearch.client.RequestOptions;
-import org.elasticsearch.index.query.BoolQueryBuilder;
-import org.elasticsearch.index.query.QueryBuilders;
+import org.elasticsearch.index.query.*;
 import org.elasticsearch.search.SearchHit;
 import org.elasticsearch.search.builder.SearchSourceBuilder;
 
-import java.io.IOException;
-import java.util.*;
-import java.util.stream.Collectors;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.*;
 
-import static org.alliancegenome.core.config.Constants.ES_INDEX;
+import lombok.extern.jbosslog.JBossLog;
 
+@JBossLog
```

Comment:
These auto generate a logger so that you can `log.info("Message");` anywhere in the code. We also DO NOT use system.out.println(); as those are not captured in the logs.